mfrc522/MFRC522.py

```diff
@@ -214,6 +214,7 @@ def MFRC522_ToCard(self, command, sendData):
 
         i = 2000
         while True:
+            time.sleep(0.25)
```

Comment:
I tried adding this to my local version of the code, and RFID reads have stopped working. CPU load is certainly a problem for me (loads of over 100 on a raspberry pi zero w), but this doesn't seem like a working fix to me?

Reply:
same here